### ISSUE TITLE
Decide on SHA package

### ISSUE THREAD

**Issue by edsko** [well-typed/hackage-security#36]
- [x] Decide on SHA package -- Done, using `cryptohash`
- [x] Update `hackage-security` to use new package
- [ ] In `hackage-security` we avoid loading files into memory -- separate ticket #134
- [ ] Update `hackage-server` to use the new package -- separate ticket #135
- [ ] In `hackage-server` we make sure these SHAs are fully evaluated -- separate ticket #136

**Comment by edsko** [comment]
hvr suggests http://hackage.haskell.org/package/cryptohash

**Comment by edsko** [comment]
Downside of `cryptohash`: has additional dependency. Needs discussion before we can merge with HEAD.

**Comment by edsko** [comment]
Choice of `cryptohash` okay; opened separate tickets about remaining items.
